ClickToBuild.cmd

```diff
@@ -1,15 +1,26 @@
-for /f "usebackq tokens=*" %%i in (`lib\vswhere\vswhere -latest -version "[15.0,16.0)" -requires Microsoft.Component.MSBuild -property installationPath`) do (
```

Comment:
Changes in this file are really needed? It looks that are not really related with thi topic of the PR

Reply:
Well, the topic of the PR in reality is very open

Reply:
I made this branch from master. And this change is a head commit to 1.10.x by sebastienros on master branch. 
And master is a stable branch. So I think I'd better submit this PR to another one.